projects/spidermonkey/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER agaynor@mozilla.com
+RUN apt-get update && apt-get install -y \
+  mercurial \
+  autoconf2.13 \
+  yasm
+
+RUN hg clone https://hg.mozilla.org/mozilla-central/
```

Comment:
can we clone from the git repo instead with --depth=1? it seems much faster.

Reply:
https://github.com/mozilla/gecko-dev ?

Reply:
yep!